scripts/releaseUtils.ts

```diff
@@ -9,7 +9,9 @@ export function run<EO extends ExecaOptions>(
   bin: string,
   args: string[],
   opts?: EO,
-): ResultPromise<EO & (keyof EO extends 'stdio' ? {} : { stdio: 'inherit' })> {
+): ResultPromise<
+  EO & (keyof EO extends 'stdio' ? object : { stdio: 'inherit' })
+> {
```

Comment:
https://v8--typescript-eslint.netlify.app/rules/ban-types: this is now covered by [`@typescript-eslint/no-empty-object-type`](https://v8--typescript-eslint.netlify.app/rules/no-empty-object-type). This particular object is better described as `object`.